### ISSUE TITLE
Create basic Genesis child theme that works with Elementor throughout - feature request

### ISSUE THREAD

**Issue by HPGHEL** [pojome/elementor#2392]
I like to use Genesis for some of its security and other features. They always recommend using a child theme, so I then get one of these. I am only doing websites to help out some friends, not as a business and don't know how to create my own child theme. However, since finding Elementor, I miss being able to edit the front page in the same way as I edit the rest of the website. In the future I'd love to be able to put a child theme on that still makes use of the Genesis framework (and any associated plugins/widgets), but is essentially blank beyond that - just the right coding to allow me to use Elementor on all aspects of design (including menu and front page).  Or if this isn't possible, to have a plain child theme with header and footer and allows for Elementor for the rest of the front page. If I could just add this to your list of considerations. I realise it may have a separate cost to it as other child themes do.  I also realise it may be better to request from Studiopress - so I could send the request to them - but I think it is more likely to appeal to people already using Elementor than their customers.

**Comment by SniffleValve** [comment]
HPGHEL - if you have and are using Genesis, then why not just use the Genesis Sample Theme (available from your Genesis account). It's about as basic as they come :) 

Now, about the ability to edit the "front page" as the rest of the site. Keep in mind that by default WP shows your latest posts on the home (front) page. If indeed that is what you wish to edit with Elementor, you cannot. Nor out of the box with any other page builder. That's because it is not a WP "page", it is a post archive listing. There are, however, plugins available for Elementor that will let you do so. The one that I use is AnyWhere Elementor Pro, which is a paid plugin. 

Cheers!
Lyle

**Comment by HPGHEL** [comment]
Hi Lyle
Thanks for that. I wasn't aware of a provided child theme with the framework as I've added premium ones.  I'll have a look in my Studiopress account and play with that then next time I'm making a site.  Generally I've done sites that don't have posts on the front page (the blog may not used at all) but more of a normal website home page.  I just wanted the flexibility of Elementor to be available on the home page not just the other pages I create.  I did find a way around it with my last website and added in a bit of code I found to the PHP file so that Elementor could work there - but it took me a whole day of moving this code around to get it to work as I have no idea how PHP works.  I thought if there was a super simple child theme, that already provided this code, that would be worth having (and paying a small amount for).  

As an even more advanced option I'd like to be able to use the new Elementor menus and create even the header with it, but when I've tried to look that up it seemed very complicated and so that may not be the best option.  I'll have a look at the plugin you mention as perhaps that could do it.

Thanks for your time.
